opm/parser/eclipse/EclipseState/Grid/FieldPropsManager.hpp

```diff
@@ -42,10 +43,14 @@ class FieldPropsManager {
     FieldPropsManager() = default;
     FieldPropsManager(const Deck& deck, const Phases& ph, const EclipseGrid& grid, const TableManager& tables);
     virtual void reset_actnum(const std::vector<int>& actnum);
+    virtual void applyNumericalAquifer(const AquiferConfig&);
     const std::string& default_region() const;
     virtual std::vector<int> actnum() const;
     virtual std::vector<double> porv(bool global = false) const;
 
+    // TODO: golbal?
+    const std::vector<double>& cellDepth() const;
```

Comment:
Do you *really* need this?

Reply:
To be honest, NO. Because the cell_depth in `FieldProps` is NOT used for anything in `Flow`, but it is something found later. 

In the simulator code, the depth is calculated based on averaging of the corner points.

Reply:
> In the simulator code, the depth is calculated based on averaging of the corner points.

Also this value is based on the average of the corner points (in the EclipseGrid) - it is just cached here for faster initiialization of the saturation properties.

Reply:
Are you sure it is used somewhere? I did not find it.

Reply:
Okay. It looks like it is used in ```SatfuncPropertyInitializers.cpp`.